site/content/en/docs/Installation/Install Agones/yaml.md

```diff
@@ -19,6 +19,23 @@ kubectl create namespace agones-system
 kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/install/yaml/install.yaml
 ```
 
+{{% feature publishVersion="1.7.0" %}}
```

Comment:
I don't think we need to feature block this - since we've already released all the helm3 features - it can go live straight away I think.